### PR TITLE
Test dev release auto

### PR DESCRIPTION
**Pull request by julian-klode**
Depends on/Incorporates #157

### CODE REVIEW

File: test/test_dev_release.py [hunk 1]
```diff
@@ -21,19 +21,35 @@ class MockOptions(object):
...
+        unattended_upgrade.LOCK_FILE = "./u-u.lock"
         self.rootdir = os.path.abspath("./root.untrusted")
         self.log = os.path.join(
             self.rootdir, "var", "log", "unattended-upgrades",
             "unattended-upgrades.log")
 
+        self.apt_conf = os.path.join(self.rootdir, "etc", "apt",
+                                     "apt.conf")
+
+        os.rename(self.apt_conf, self.apt_conf + ".bak")
```
Comment (rbalint) [inline test/test_dev_release.py]: would not it be nicer to use a fragment in apt.conf.d/ ?

Comment (julian-klode) [reply test/test_dev_release.py]: I don't know, I feel like we should do some generic refactoring of the test rootdir interaction.

Comment (rbalint) [reply test/test_dev_release.py]: well, i share the feeling then :-)
it is fine for now

**Comment by rbalint** [comment]
@julian-klode Merging was a bit early, autopkgtest fails in unstable and disco when building from source :-(

**Pull request merged**